devops/azure-pipelines/build-stage-template.yml

```diff
@@ -0,0 +1,17 @@
+stages:
+- stage: Build
+  displayName: ${{ parameters.displayName }}
+  jobs:
+  - job: Build
+    steps:
+    - task: NodeTool@0
+      inputs:
+        versionSpec: '10.x'
```

Comment:
node 10 is deprecated. use node 14 or 12, one of the maintained long-term-support versions. might as well use 14 if nothing breaks with it (test this)